### PR TITLE
Moved login to homepage

### PR DESCRIPTION
**Pull request by cmcahoon01**
Now toggles alerts and submit feedback when logged in

### CODE REVIEW

**Comment by cmcahoon01** [comment]
Animation2

**Review by abijeet-dhillon — DISMISSED** [review]
I created a test user and it worked for me, great job! Just need to fix the merge conflicts.

**Review by abijeet-dhillon — CHANGES_REQUESTED** [review]
I created a test user and it worked for me, great job! Just need to fix the merge conflicts.

File: Site/bc_weather_station_dashboard/website/templates/home.html [hunk 1]
```diff
@@ -231,6 +286,12 @@ <h4 class="modal-title">Submit Feedback</h4>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js"
         integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
         crossorigin="anonymous"></script>
+    <script>
+        {% if error %}
```
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/templates/home.html]: whoops didn't mean to make a suggestion just wanted to highlight that, did that in vscode so thats why. mb

Comment (cmcahoon01) [reply Site/bc_weather_station_dashboard/website/templates/home.html]: I'm not sure. My IDE doesn't say anything, but if yours has a suggestion feel free

**Review by robertyac — CHANGES_REQUESTED** [review]
Looks good and works as intended for me!!

One thing, we may want to add a field in the register user that they can choose a user name. Just so we can then display their username in the nav bar in the top right corner instead of a profile picture, and perhaps when they click that the dropdown still appears? 

 since right now user name is defaulted to their email:
Screenshot 2024-03-07 at 5 13 02 PM

LMK what you think

This can also be another issue if need be...

**Comment by cmcahoon01** [comment]
Yeah, I thought about that and decided I wished I had a manager or design specialist to decide for me. It would be a trivial change, but it would make registration a bit bulkier so I left it at just username for now. Other thoughts?

**Comment by robertyac** [comment]
Yeah that's fair, I just thought it would be good for users to create their own username since we could use that instead of a profile image but it's not a huge deal just depends what everyone thinks

**Comment by carsondrobe** [comment]
Works, will need to make an issue to link this user to their Feedback

**Review by carsondrobe — APPROVED** [review]
LGTM, will add an issue for linking these Users to Feedback View

**Review by abijeet-dhillon — APPROVED** [review]
Works correctly for me, great job!

**Review by robertyac — APPROVED** [review]

**Review by robertyac — CHANGES_REQUESTED** [review]
Add username field (I take back my approval)

**Review by robertyac — APPROVED** [review]

**Comment by robertyac** [comment]
Works well, tested it out. nice work

**Pull request merged**